src/apollo/client.js

```diff
@@ -4,7 +4,7 @@ import { HttpLink } from 'apollo-link-http'
 
 export const client = new ApolloClient({
   link: new HttpLink({
-    uri: 'https://api.thegraph.com/subgraphs/name/uniswap/uniswap-v2',
+    uri: 'http://3.22.187.59/subgraphs/name/fuseio/fuseswap',
```

Comment:
Should i change these to env params?

Reply:
Don't think we need to introduce more changes here. These values should be are used only once here, so it'll be easy to change them

Reply:
Cool